.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a bug report
+title: ''
+labels: bug, needs triage
+assignees: ''
+
+---
+
```

Comment:
I would also add a note here about first checking for existing issues that. It could be a comment that doesn't necessarily show up as text

```
<!--- Before opening up a new bug report, please make sure to check for similar existing issues -->
```